src/core/utils.js

```diff
@@ -107,5 +107,87 @@ const resolvePath = promisify(function (objectAPI, ipfsPaths, callback) {
   }, callback)
 })
 
+/**
+ * Parses chunker string into options used by DAGBuilder in ipfs-unixfs-engine
+ *
+ *
+ * @param  {String}   chunker Chunker algorithm supported formats:
+ *                    "default" ("")
+ *                    "size-{size}",
+ *                    "rabin"
+ *                    "rabin-{avg}"
+ *                    "rabin-{min}-{avg}-{max}"
+ *
+ * @return {Object}   Chunker options for DAGBuilder
+ */
+function parseChunkerString (chunker) {
+  if (!chunker || chunker === '') {
```

Comment:
Do we ever pass in an empty string (or `undefined`)? If not, this if case could be removed (and also removed from the JSDocs).

Reply:
Looks safe to remove